bin/dev-dump-program-config

```diff
@@ -0,0 +1,31 @@
+#! /usr/bin/env bash
+
+# DOC: Dumps question and program configuration from the local dev environment into postgres archive file, dev_programs.dump.
+
+source bin/lib.sh
+docker::set_project_name_dev
+readonly COMPOSE_CMD="${DOCKER_COMPOSE_DEV}"
+readonly DUMPFILE="${1:-dev_programs.dump}"
+
+readonly CMD="/usr/bin/pg_dump \
+    -w \
+    -Fc \
+    -h db \
+    -U postgres \
+    -t programs \
+    -t questions \
+    -t versions \
+    -t versions_programs \
+    -t versions_questions \
+    postgres -f /project/${DUMPFILE}"
+
+${COMPOSE_CMD} \
```

Comment:
just a note that this i think is supposed to be $() in a subshell and all in quotes according to the style guide? as far as i can tell but it's not super clear on putting commands in variables

Reply:
Not in this case - We're saving a command and some of it's arguments to a variable, so quoting it here will prevent them from splitting into different values and correctly getting run.